packages/vue-app/template/components/nuxt.js

```diff
@@ -12,12 +12,15 @@ import NuxtError from './nuxt-error.vue'
 <% } %>
 import NuxtChild from './nuxt-child'
 
+<%if (isDev) { %>import NuxtHMR from './nuxt-hmr'<% } %>
```

Comment:
Also wrapped with `client` ?

Reply:
This component is SSR safe and renders an empty node (only for dev)